### PR TITLE
Fix for vimeo playback issues brave/brave-browser#4770

### PR DESCRIPTION
**Pull request by ryanbr**
Allow vimeo to work on various sites. 

```
https://community.brave.com/t/vimeo-cookies-sorry-because-of-its-privacy-settings-this-video-cannot-be-played-here/61505/6
https://community.brave.com/t/not-able-to-view-vimeo-videos/48706
https://community.brave.com/t/vimeo-private-videos-not-showing/44325
```

### CODE REVIEW

**Comment by bbondy** [comment]
@diracdeltas are you ok with these?

**Comment by diracdeltas** [comment]
lgtm if these work and fix the issue. @ryanbr can you please start a wiki doc in https://github.com/brave/referrer-whitelist/wiki documenting why each of these exceptions was added? i will add explanations for the ones i know about too.

**Comment by diracdeltas** [comment]
@ryanbr do you have a test plan we could use to verify this fix? for instance i tried going to https://ultimatecourses.com/learn/javascript-basics but that video shows up as not-playable either way. (it's only playable if you allow all cookies)

File: data/ReferrerWhitelist.json [hunk 1]
```diff
@@ -4,7 +4,9 @@
             "<all_urls>": [
                 "https://use.typekit.net/*",
                 "https://api.geetest.com/*",
-                "https://cloud.typography.com/*"
+		"https://cloud.typography.com/*",
+		"https://*.vimeocdn.com/*",
+		"https://player.vimeo.com/*"
```
Comment (diracdeltas) [inline data/ReferrerWhitelist.json]: could you please indent using spaces instead of tabs here?

Comment (ryanbr) [reply data/ReferrerWhitelist.json]: sure. Done.

**Comment by ryanbr** [comment]
was under the impression that it was a referral issue (which caused the cookie fault issue), and sure enough if I via scriptsafe extension, and only allow the above elements it should work. I'm open a better suggestion or fix.  

If its definitely not a referrer issue causing this, just close it.

**Comment by ryanbr** [comment]
Sorry, still learning the ropes the systems. But here is a gif, of the fix in action; 
https://secure.fanboy.co.nz/gif/vimeo2.gif

**Comment by ryanbr** [comment]
Is that okay @diracdeltas ?

**Review by diracdeltas — APPROVED** [review]
i must have tested something incorrectly before since https://ultimatecourses.com/learn/javascript-basics works now with this fix.

@pilgrim-brave feel free to squash these commits and merge

**Comment by pilgrim-brave** [comment]
Everything is de-conflicted and merged. Testing locally to ensure I didn't break anything, then I'll run the Jenkins job to push the changes to production.

**Pull request merged**